catalyst_voices/packages/internal/catalyst_voices_models/lib/src/campaign/campaign_publish.dart

```diff
@@ -0,0 +1,15 @@
+// Enum representing the state of a campaign.
+// Draft: campaign is not published yet.
+// Published: campaign is published and can be seen by users.
+//
+// This enum is used in conjunction with the CampaignStage enum
+// from view_models/campaign/campaign_stage.dart to determine
+// the stage of a campaign.
+//
+// If CampaignPublish is draft then CampaignStage must be draft.
```

Comment:
```suggestion
// If [CampaignPublish] is draft then [CampaignStage] must be draft.
```

Make sure to also import it. This way if you rename a class your IDE will rename the comment too

Reply:
models don't have access to view_models I belive.

Reply:
hmm, wouldn't it make sense to move this doc into view_models so that imports can be solved? Shouldn't make sense to document things in a package from where you cannot access them

Reply:
Good idea!